chart/compass/values.yaml

```diff
@@ -958,6 +958,11 @@ global:
       certValidity: "7"
       clientCertRetryAttempts: "8"
       containerName: "certificate-rotation"
+    preInstallJob:
+      enabled: false
```

Comment:
General question: as there are no overrides for the benchmark installation, do we want to run these pre jobs there or not?

The benchmark is closer to a local setup but on the other hand it is a clean installation on a cluster so that raised my question.